app/models/runtime/asg_latest_update.rb

```diff
@@ -1,6 +1,7 @@
 module VCAP::CloudController
   class AsgLatestUpdate
     class AsgTimestamp < Sequel::Model
+      plugin :microsecond_timestamp_precision
```

Comment:
@Gerg had mentioned that there was a [request for another field to support fractional seconds](https://github.com/cloudfoundry/cloud_controller_ng/blob/e11485f8b6737cf781e784b92bce7271c6f26aea/app/jobs/diego/sync.rb#L24-L27) as well, so I went with the modular approach here so it could be reused, rather than having to refactor it out of here when support for that field comes along.